app/frontend/src/components/bcgov/BCGovFooter.vue

```diff
@@ -55,5 +61,12 @@ export default {
     text-decoration: none;
     text-transform: none;
   }
+
+  @media print {
+    &.no-print,
+    &.no-print ::v-deep * {
+      display: none !important;
+    }
+  }
```

Comment:
Do the Vuetify built in classes work here instead?
https://vuetifyjs.com/en/styles/display/#display-in-print

Reply:
that's great. that class does work.

Reply:
Oh awesome, haven't had a chance to try those so far.